javascript/javascript1/week2/candyfunction.js

```diff
@@ -0,0 +1,40 @@
+let boughtCandyPrices = [];
+function addCandy(candyType,weight){
+    if(candyType = "sweet"){
```

Comment:
One equal means assigning the value to the variable. 
- Use `==` to compare values
- Or use `===` to compare both value and type *Recommended Always*